sql/agtype_gin.sql

```diff
@@ -70,6 +70,8 @@ DEFAULT FOR TYPE agtype USING gin AS
   OPERATOR 9 ?(agtype, agtype),
   OPERATOR 10 ?|(agtype, agtype),
   OPERATOR 11 ?&(agtype, agtype),
+  OPERATOR 12 @>>(agtype, agtype),
+  OPERATOR 13 <<@(agtype, agtype),
```

Comment:
@jrgemignani any idea how I should be adding this change to the template file?

Reply:
The template file is basically the sql commands that are necessary to modify the previous (current) installed version to the next (new) version. So, if you are adding something, then just the sqls to add it. Basically, the sql for the age*.sql file. If you were modifying something existing, changing it, then the sql to modify the previous version to the new version. If you are removing something, then the sql to remove it.

Hopefully, that helps.

Reply:
Hi @MuhammadTahaNaveed Where you able to set it up?

Reply:
> Hi @MuhammadTahaNaveed Where you able to set it up?

I couldn't find any option to add an operator in operator class using ALTER command https://www.postgresql.org/docs/current/sql-alteropclass.html, so I dropped and recreated the operator class in that template file.

I have pushed these changes, please review at your convenience.

Reply:
You should just be able to add the commands that were added to the split age--x.x.x.sql file?

Reply:
@MuhammadTahaNaveed Oh, I see what you mean now